File: c++/iniciante/1564.cpp
#include <iostream>

int main() {
    int n;
    while (std::cin >> n) {
        const char* out = "vai ter copa!";
        if (n > 0) {
            out = "vai ter duas!";
        }
        std::cout << out << std::endl;
    }
    return 0;
}

File: go/iniciante/1564.go
package main

import "fmt"

func main() {
	var n int
	for {
		if _, err := fmt.Scanf("%d", &n); err != nil {
			break
		}

		out := "vai ter copa!"
		if n > 0 {
			out = "vai ter duas!"
		}
		fmt.Println(out)
	}
}
